src/pipecat/services/azure.py

```diff
@@ -61,6 +61,67 @@
     raise Exception(f"Missing module: {e}")
 
 
+def language_to_azure_language(language: Language) -> str | None:
+    language_map = {
+        Language.BG: "bg-BG",
+        Language.CA: "ca-ES",
+        Language.ZH: "zh-CN",
+        Language.ZH_TW: "zh-TW",
+        Language.CS: "cs-CZ",
+        Language.DA: "da-DK",
+        Language.NL: "nl-NL",
+        Language.EN: "en-US",
+        Language.EN_US: "en-US",
+        Language.EN_AU: "en-AU",
+        Language.EN_GB: "en-GB",
+        Language.EN_NZ: "en-NZ",
+        Language.EN_IN: "en-IN",
+        Language.ET: "et-EE",
+        Language.FI: "fi-FI",
+        Language.NL_BE: "nl-BE",
+        Language.FR: "fr-FR",
+        Language.FR_CA: "fr-CA",
+        Language.DE: "de-DE",
+        Language.DE_CH: "de-CH",
+        Language.EL: "el-GR",
+        Language.HI: "hi-IN",
+        Language.HU: "hu-HU",
+        Language.ID: "id-ID",
+        Language.IT: "it-IT",
+        Language.JA: "ja-JP",
+        Language.KO: "ko-KR",
+        Language.LV: "lv-LV",
+        Language.LT: "lt-LT",
+        Language.MS: "ms-MY",
+        Language.NO: "nb-NO",
+        Language.PL: "pl-PL",
+        Language.PT: "pt-PT",
+        Language.PT_BR: "pt-BR",
+        Language.RO: "ro-RO",
+        Language.RU: "ru-RU",
+        Language.SK: "sk-SK",
+        Language.ES: "es-ES",
+        Language.SV: "sv-SE",
+        Language.TH: "th-TH",
+        Language.TR: "tr-TR",
+        Language.UK: "uk-UA",
+        Language.VI: "vi-VN",
+    }
+    return language_map.get(language)
+
+
+def sample_rate_to_output_format(sample_rate: int) -> SpeechSynthesisOutputFormat:
+    sample_rate_map = {
+        8000: SpeechSynthesisOutputFormat.Raw8Khz16BitMonoPcm,
+        16000: SpeechSynthesisOutputFormat.Raw16Khz16BitMonoPcm,
+        22050: SpeechSynthesisOutputFormat.Raw22050Hz16BitMonoPcm,
+        24000: SpeechSynthesisOutputFormat.Raw24Khz16BitMonoPcm,
+        44100: SpeechSynthesisOutputFormat.Raw44100Hz16BitMonoPcm,
+        48000: SpeechSynthesisOutputFormat.Raw48Khz16BitMonoPcm,
+    }
+    return sample_rate_map.get(sample_rate, SpeechSynthesisOutputFormat.Raw16Khz16BitMonoPcm)
```

Comment:
Should the default be the 24khz option since that's now the default?

```suggestion
    return sample_rate_map.get(sample_rate, SpeechSynthesisOutputFormat. Raw24Khz16BitMonoPcm)
```